tests/brun/path-6.txt

```diff
@@ -1,3 +1,3 @@
 brun -n -c '56800' '((((((0x1337 . (0x1337 . (0x1337 . (0x1337 . ((0x1337 . (0x1337 . (0x1337 . ((0x1337 . 42) . 0x1337)))) . 0x1337))))) . 0x1337) . 0x1337) . 0x1337) . 0x1337) . 0x1337)'
-cost = 68
+cost = 69
```

Comment:
it might be unintuitive at first, why these costs changed. However, keep in mind that since the path is entered as a positive decimal number, and the most significant bit is set, we implicitly add a leading zero to ensure it's not interpreted as a negative number. It would be more efficient to specify this path as `0xDDE0`, since it would not have a leading zero in that case.

Reply:
You might consider adding a test that does just that (`0xdde0` instead of `56800`) .

Reply:
there are tests for that too. path-9 and path-10 iirc

Reply:
nooo. I must have forgotten `git add path-10.txt` :/